README.md

```diff
@@ -0,0 +1,70 @@
+# Metrics Webapp
+
+>A mobile web application to check a list of metrics (numeric values) about covid 19 data for 2020-03-22 using the narrativa api. This API for the app all gets numeric values based on a parameter. This project was built using React and Redux.
+
+## Screenshots
+
+<img src="./src/assets/images/Screenshot 2022-06-01 at 9.12.11 AM.png">
+<img src="./src/assets/images/Screenshot 2022-06-03 at 12.57.43 PM.png">
+<img src="./src/assets/images/Screenshot 2022-06-03 at 12.58.59 PM.png">
+
+## Built With
+
+- Technologies used:
+  - React
+  - Html
+  - CSS
+  - Jest
+  - Redux
+
+## Live preview
+
+ [click here](https://fantastic-shortbread-de13a8.netlify.app)
+
+## Video presentation
+
+ [click here](https://www.loom.com/share/b3efdda77cd5494892d60420a6897647)
+
```

Comment:
- [ ] As pointed out by the previous reviewer kindly record another video, a maximum of 5 minutes, in that video don't forget to highlight the following
  - A description of the project.
  - A demo of the project features.
  - You should also highlight some interesting piece of code or something you built that you are very proud of.

Reply:
This is another video

Reply:
hi @Gnwin, one of the requirements is that the video should be 5 minutes max. I see that this one is 11 minutes. Kindly fix that.